### PR TITLE
Revise README. (read extended commit message)

### PR DESCRIPTION
**Pull request by ckiee**
* add Arch Linux instructions
* add instructions for other OS'
* move NixOS instructions

P.S: I made the AUR package (:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
 
+## Installing & Running
+
+### Arch Linux
+Use your favorite AUR helper and install the `jumpcutter-git` package.
+
+For example, Using yay: `yay jumpcutter-git`
+Then you can use the binary located in /usr/bin/jumpcutter
+
+### NixOS
```
Comment (balsoft) [inline README.md]: Please rename back to nix. NixOS is a linux distro, Nix is a package manager. Nix can be used on other distros as well.

**Comment by balsoft** [comment]
Could you rename NixOS section back to Nix?

**Review by balsoft — CHANGES_REQUESTED** [review]
Could you rename NixOS section back to Nix?

**Comment by nevercast** [comment]
Can you link to the AUR repo too

File: README.md (hunk 1 above)
Comment (CoolOppo) [reply README.md]: Or how about "Nix Package Manager" as that disambiguates whether you mean the package manager or simply "all *nix distros".

Comment (RONTheCookie) [reply README.md]: Done (:

**Comment by ckiee** [comment]
Cleaning my site-wide GitHub PR list.

**Pull request closed without merging**